src/main/fc/fc_core.c

```diff
@@ -912,11 +912,7 @@ static void subTaskMotorUpdate(timeUs_t currentTimeUs)
 
 uint8_t setPidUpdateCountDown(void)
 {
-    if (gyroConfig()->gyro_soft_lpf_hz) {
-        return pidConfig()->pid_process_denom - 1;
-    } else {
-        return 1;
-    }
+    return pidConfig()->pid_process_denom - 1;
```

Comment:
Change looks ok, wondering if we should drop the function, since it does not improve the readability (see case at hand).

Reply:
I'm in favor of dropping it, but we could offset it for 3.4.

Reply:
No reason not to drop it as part of the fix - there is no additional risk.

'I'll make it pretty in the next version' are pretty much every developers famous last words...